src/utils/bititer.h

```diff
@@ -35,20 +35,40 @@ namespace lczero {
 
 inline unsigned long GetLowestBit(std::uint64_t value) {
 #if defined(_MSC_VER) && defined(_WIN64)
-    unsigned long result;
-    _BitScanForward64(&result, value);
-    return result;
+  unsigned long result;
+  _BitScanForward64(&result, value);
+  return result;
 #elif defined(_MSC_VER)
-    unsigned long result;
-    if (value & 0xFFFFFFFF) {
-      _BitScanForward(&result, value);
-    } else {
-      _BitScanForward(&result, value >> 32);
-      result += 32;
-    }
-    return result;
+  unsigned long result;
+  if (value & 0xFFFFFFFF) {
+    _BitScanForward(&result, value);
+  } else {
+    _BitScanForward(&result, value >> 32);
+    result += 32;
+  }
+  return result;
 #else
-    return __builtin_ctzll(value);
+  return __builtin_ctzll(value);
+#endif
+}
+
+inline unsigned long GetLowestBit32(std::uint32_t value) {
+#if defined(_MSC_VER)
+  unsigned long result;
+  return _BitScanForward(&result, value);
```

Comment:
No return here.

Reply:
Applied the patch that you suggested (with some changes).